utils/validators.js

```diff
@@ -0,0 +1,11 @@
+function validateEmail(email) {
```

Comment:
use validator npm package for email validation/other validations: (https://www.npmjs.com/package/validator)